news/2023/2023-06-02-the-osu-legacy-of-reesewashere.md

```diff
@@ -0,0 +1,45 @@
+
+Many iconic plays that have forever been engraved into osu! history were captured during this period, such as the [1st ever FC on IMAGE -MATERIAL-](https://www.youtube.com/watch?v=Vm-zilvu8F0), the [1st ever FC on Everything will freeze](https://www.youtube.com/watch?v=5d432cQoT-c) and the [1st ever 700pp score on Blue Zenith](https://www.youtube.com/watch?v=UYNpkDrCWtA), from which "727" meme was born and still refuses to die to this day ([thanks Aireu](https://www.youtube.com/watch?v=AaAF51Gwbxo&t=72s)).
+
+Unknown to him in the moment, Reese's YouTube channel was starting to turn into a time capsule of a simpler era, what some commentators today consider to be the "golden age of osu!". At the time, the combination of the hype towards fierce competition for the #1 spot, a legendary player returning and the mapping meta shifting to "pp mapping" (allowing for *big* pp numbers) was all spurring the community into a score-craving frenzy. 
+
+High-achieving players such as [Rafis](https://osu.ppy.sh/users/2558286), [Angelsim](https://osu.ppy.sh/users/1777162) and Cookiezi would all log on to a sea of viewers and spectators. Foundations were laid for practices like "replay recording", paving the way for YouTube channels such as [Circle People](https://www.youtube.com/@CirclePeople) to pick up the torch later in 2016, and spurning a period of fierce competition between channels fighting to be the first to snatch the latest big play.
+
+But what the community didn't know is that Reese's uploads were about to come to a screeching halt.
+
+![](/wiki/shared/news/2023-06-02-osu-legacy-of-reesewashere/tweet.png)
+
+On January 30th 2016, Reese officially announced via a now deleted [Reddit post](https://www.reddit.com/r/osugame/comments/43c2pk/future_of_the_reesewashere_youtube_channel/) on r/osugame and a [tweet](https://twitter.com/ReeseWasHere/status/693250557419593729) that the channel will no longer be active. I asked him if there was anything else that affected his decision to quit, besides getting a full-time job:
+
+> ...after I graduated and started working I didn't really have much time to play osu! anymore, and I especially didn't have time to record liveplays. I wasn't really into the idea of just uploading videos of replay files, and I saw other channels were filling that gap, so I just stopped uploading.
+
+To my question of whether he still keeps up with the community, he responded that he participated in r/place last year and has kept up with the development of osu!(lazer), and is eagerly waiting for it to be the new main client. It was suggested that he will return at some point, but that the contributions would more likely be via coding, rather than uploading videos once again.
```

Comment:
```suggestion
To the question of whether he still keeps up with the community, he responded that he participated in r/place last year and has kept up with the development of osu!(lazer), eagerly waiting for it to be the new main client. He also suggested that he may return as a player at some point, but that any more contributions would likely happen via code, rather than uploading videos once again.
```